docs/subnets/customize-a-subnet.md

```diff
@@ -577,6 +575,140 @@ This example enables the `feeManagerConfig` at the first block with timestamp >=
 
 When a precompile disable takes effect (ie., after its `blockTimestamp` has passed), its storage will be wiped. If you want to reenable it, you will need to treat it as a new configuration.
 
+### eth_getChainConfig
```

Comment:
Can we add to the C-Chain config file documentation as well to note which API name needs to be enabled to enable these two API calls?

Reply:
added. could you check?

Reply:
Looking at this again, this doesn't really fit into this file currently. Can we add a header for these two so that they fit into this file better? We could do `Verifying the Chain Config` and put a small blurb about the purpose of these APIs.